go.mod

```diff
@@ -14,9 +14,11 @@ require (
 )
 
 require (
-	github.com/google/renameio/v2 v2.0.0 // indirect
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
-	github.com/mdlayher/watchdog v0.0.0-20201005150459-8bdc4f41966b // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
-	rsc.io/goversion v1.2.0 // indirect
 )
+
+// TODO: Remove before merging
```

Comment:
BTW: You might be interested in giving Go workspaces a shot to get rid of per-repository go.mod replace directives? https://utcc.utoronto.ca/~cks/space/blog/programming/GoWorkspacesComing

Haven’t used them myself, though :)

Reply:
Oh yeah, its not something I've tried out yet either.